chutzpah/Samples/readme.txt

```diff
@@ -1,26 +0,0 @@
-Chutzpah Samples 
```

Comment:
Yeah you're right! I'll make a copy of this file so we use for future reference.